src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5146,11 +5147,16 @@ bool CEndlessUsbToolDlg::ExtendImageFile(const CString &endlessImgPath, ULONGLON
 	LARGE_INTEGER lsize;
 	lsize.QuadPart = selectedGigs * BYTES_IN_GIGABYTE;
 	
-	uprintf("Trying to extend Endless image from %I64i bytes to %I64i bytes which is about %s", lCurrentSize.QuadPart, lsize.QuadPart, SizeToHumanReadable(lsize.QuadPart, FALSE, use_fake_units));
-	IFFALSE_GOTOERROR(SetFilePointerEx(endlessImage, lsize, NULL, FILE_BEGIN) != 0, "Error on SetFilePointerEx");
-	IFFALSE_GOTOERROR(SetEndOfFile(endlessImage), "Error on SetEndOfFile");
-	IFFALSE_GOTOERROR(SetFileValidData(endlessImage, lsize.QuadPart), "Error on SetFileValidData");
-	uprintf("Extended Endless image");
+    if (lCurrentSize.QuadPart < lsize.QuadPart) {
```

Comment:
During testing when I hadn't implemented reading the uncompressed size, I just set it to 1MB.

Afaict it will not happen in practice but why risk it!